Libplanet.Explorer/ExplorerStartup.cs

```diff
@@ -44,7 +41,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<IBlockChainContext<T>, TU>();
             services.AddSingleton<IStore>(
-                services => services.GetService<IBlockChainContext<T>>().Store);
+                provider => provider.GetService<IBlockChainContext<T>>()!.Store);
+            services.AddSingleton<IBlockChainIndex>(
+                provider => provider.GetService<IBlockChainContext<T>>()!.Index);
```

Comment:
If it expects that `IBlockChainContext<T>` must exist, how about using [`ServiceProviderServiceExtensions.GetRequiredService<T>(...)`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.serviceproviderserviceextensions.getrequiredservice?view=dotnet-plat-ext-7.0) method?
```suggestion
                provider => provider.GetRequiredService<IBlockChainContext<T>>().Store);
            services.AddSingleton<IBlockChainIndex>(
                provider => provider.GetRequiredService<IBlockChainContext<T>>().Index);
```